common/shlibs

```diff
@@ -2924,15 +2924,17 @@ libdecoration.so.0 compiz-core-0.8.14_1
 libcompizconfig.so.0 libcompizconfig-0.8.14_1
 libemeraldengine.so.0 emerald-0.8.14_1
 libhangul.so.1 libhangul-0.1.0_1
-libmutter-7.so.0 mutter-3.37.90_1
-libmutter-clutter-7.so mutter-3.37.90_1
-libmutter-clutter-7.so.0 mutter-3.37.90_1
-libmutter-cogl-path-7.so mutter-3.37.90_1
-libmutter-cogl-path-7.so.0 mutter-3.37.90_1
-libmutter-cogl-7.so mutter-3.37.90_1
-libmutter-cogl-7.so.0 mutter-3.37.90_1
-libmutter-cogl-pango-7.so mutter-3.37.90_1
-libmutter-cogl-pango-7.so.0 mutter-3.37.90_1
+libmutter-8.so.0 mutter-40.0_1
+libmutter-clutter-8.so mutter-40.0_1
+libmutter-clutter-8.so.0 mutter-40.0_1
+libmutter-cogl-8.so mutter-40.0_1
+libmutter-cogl-8.so.0 mutter-40.0_1
+libmutter-cogl-pango-8.so mutter-40.0_1
+libmutter-cogl-pango-8.so.0 mutter-40.0_1
+libmutter-cogl-8.so.0 mutter-40.0_1
+libmutter-cogl-pango-8.so.0 mutter-40.0_1
+libmutter-clutter-8.so.0 mutter-40.0_1
+libmutter-8.so.0 mutter-40.0_1
```

Comment:
I'm guessing you want the `.so.0` split apart from the `.so` files but you accidentally left them in above.

Reply:
fixed